src/hmrc-design-patterns/internal-header/index.njk

```diff
@@ -23,7 +23,7 @@ layout: twoColumnPage.njk
 
 {{ hmrcFrontendInfo() }}
 
-<p class="govuk-body">The HMRC banner sits above optional elements like the <a class="govuk-link" href="https://design-system.service.gov.uk/components/service-navigation/">service navigation</a> component.</p>
+<p class="govuk-body">The HMRC banner sits above other elements like the <a class="govuk-link" href="https://design-system.service.gov.uk/components/service-navigation/">service navigation</a> component.</p>
```

Comment:
Do we need to mention this here at all? It's kind of a bit odd that we say 'How it (Internal header) works' > 'The HMRC banner...' etc.
Should services be using the Internal header and HMRC banner together? They pretty much do the same thing I would have thought.